governance.qmd

```diff
@@ -0,0 +1,117 @@
+---
+title: "R-Multiverse Governance"
+---
+
+The R-Multiverse project is run in a collaborative and open way.
+Portions of this governance policy have been inspired by that of the Apache Software Foundation.
+
+# The R-Multiverse Project
+
+The R-Multiverse project consists of the code and repositories that reside in the `r-multiverse` GitHub organization and the r-multiverse.org website.
+All repositories are open to everyone, with the exception of a private repository where security and conduct-related matters are shared between the administrators only.
```

Comment:
Maybe "All repositories are visible to everyone" might be clearer than "All repositories are open to everyone".